src/api/hotel/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+
+import { createHotelController } from "./hotel.controller";
+
+const router = Router();
+
+// /api/hotels -> POST
```

Comment:
It's not necessary comments in this kind of implementations, i.e. this endpoint.